### DISCUSSION TITLE
Multi-page website collection Blueprint

### DISCUSSION THREAD

**Discussion by Kamlesh-62** [statamic/cms#8276] [Q&A]
As a newcomer to Statamic, I'm aiming to develop a multi-section website consisting of pages such as home, product, about us, and contact us. After studying the documentation, I initially considered creating separate collections for each section, like a "home" collection with distinct entries for each section of the home page. However, upon further exploration, I discovered that all collections share the same entry blueprint.

I would greatly appreciate your advice on how to devise a blueprint for my project that accommodates multiple sections differentiated by home, product, about us, and contact us.

**Comment by ryanmitchell** [comment]
Have you looked into the "[peak](https://statamic.com/starter-kits/studio1902/peak)" starter kit? It uses a page builder approach which would give you some pointers on how you can structure this.

**Reply by Kamlesh-62** [reply]
Thanks
